lib/std/posix.zig

```diff
@@ -4412,6 +4412,18 @@ pub fn waitpid(pid: pid_t, flags: u32) WaitPidResult {
     }
 }
 
+pub fn getpid() pid_t {
+    if (@hasDecl(system, "getpid"))
+        return system.getpid();
+    @compileError("getpid() not yet implemented for target system");
```

Comment:
```suggestion
    return system.getpid();
```

`@hasDecl` is usually a code smell. Better to just the underlying compile error bubble up.